src/EdFi.AnalyticsMiddleTier.Tests/Classes/StudentAssessmentFact.cs

```diff
@@ -19,6 +19,7 @@ public class StudentAssessmentFact
         public string Namespace { get; set; }
         public string StudentAssessmentIdentifier { get; set; }
         public string StudentUSI { get; set; }
+        public string StudentKey { get; set; }
```

Comment:
We need to add this new field on the `CandidateDim` class as well.

Reply:
Done.